### PR TITLE
updated gifs

### LINKED ISSUE #39: Refresh some README gifs
Before we release v2 with more features, we should refresh some of the README gifs because more subcommands have been added since the first gifs were created, and some subcommands got more features e.g. https://github.com/ipinfo/cli/issues/22

### PR DESCRIPTION
**Pull request by talhahwahla**
replaced older gifs with newer ones. these ones show the latest info and are generated with asciinema.
Closes #39

### CODE REVIEW

File: gif/README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 This folder contains all the metadata needed to produce the `.gif` files used
 in the root README.
 
-The program used to process the metadata is
-https://github.com/faressoft/terminalizer.
+The program uses asciinema to generate .cast files and renders them with
+https://dstein64.github.io/gifcast/
+
```
Comment (rm-Umar) [inline gif/README.md:6]:
```suggestion
```

File: gif/README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 This folder contains all the metadata needed to produce the `.gif` files used
 in the root README.
 
-The program used to process the metadata is
-https://github.com/faressoft/terminalizer.
+The program uses asciinema to generate .cast files and renders them with
+https://dstein64.github.io/gifcast/
```
Comment (rm-Umar) [inline gif/README.md:5]:
``` suggestion
https://dstein64.github.io/gifcast/.
```

File: gif/README.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 This folder contains all the metadata needed to produce the `.gif` files used
 in the root README.
 
-The program used to process the metadata is
-https://github.com/faressoft/terminalizer.
+The program uses asciinema to generate .cast files and renders them with
```
Comment (rm-Umar) [inline gif/README.md:4]:
```suggestion 
The program uses `asciinema` to generate `.cast` files and renders them with

Comment (rm-Umar) [reply gif/README.md:4]: You mentioned `The program uses` but there's no program in the PR. Are you using some custom program that uses `asciinema`? if yes then mention the program also. If you are using `asciinema` to generate `.cast` files then mention it clearly.

**Review by rm-Umar — CHANGES_REQUESTED** [review]

**Review by rm-Umar — CHANGES_REQUESTED** [review]
The gifs are too big, should reduce the font size.
Links are broken in `Any IP`, `Piping` (both `cat.gif` and `cat-csv.gif`), and `Field Filter` sections. kindly update the links in the readme file or map the file names according to the readme.

**Comment by rm-Umar** [comment]
Seems good! Thank you @talhahwahla for your contribution!

**Review by rm-Umar — APPROVED** [review]

File: gif/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 This folder contains all the metadata needed to produce the `.gif` files used
 in the root README.
 
-The program uses asciinema to generate .cast files and renders them with
-https://dstein64.github.io/gifcast/
+Asciinema is used to generate the `.cast` files. These `.cast` files are rendered into
```
Comment (rm-Umar) [inline gif/README.md:4]:
```suggestion
`Asciinema` is used to generate the `.cast` files. These `.cast` files are rendered into 
```

**Review by rm-Umar — CHANGES_REQUESTED** [review]
@talhahwahla one last thing and then we are done.

**Review by rm-Umar — APPROVED** [review]

**Comment by talhahwahla** [comment]
You are welcome!

**Pull request merged**